gc3libs/etc/gcsvBulk_wrapper.m

```diff
@@ -0,0 +1,74 @@
+function gcsvBulk_wrapper(function_name, csvFile, limit, offset)
+%MATLAB_CSV_WRAPPER executes a given function with a given csv file with each line as
```

Comment:
Maybe move this comment block at the start of the file, before the `function ...` definition, so to make it clear that this is a MATLAB file and what's its purpose?